namespaces/live-1.cloud-platform.service.justice.gov.uk/pathfinder-dev/resources/rds.tf

```diff
@@ -23,21 +31,63 @@ module "dps_rds" {
   }
 }
 
+resource "aws_iam_user" "user" {
+  name = "pathfinder-rds-to-s3-snapshots-user-${random_id.id.hex}"
+  path = "/system/pathfinder-rds-to-s3-snapshots-user/"
+}
+
+resource "aws_iam_access_key" "user" {
+  user = aws_iam_user.user.name
+}
+
+data "aws_iam_policy_document" "policy" {
+  statement {
+    actions = [
+      "rds:DescribeDBClusterSnapshots",
+      "rds:DescribeDBClusters",
+      "rds:DescribeDBInstances",
+      "rds:DescribeDBSnapshots",
+      "rds:DescribeExportTasks",
+      "rds:StartExportTask",
+      "s3:PutObject*",
+      "s3:GetObject*",
+      "s3:ListBucket",
+      "s3:DeleteObject*",
+      "s3:GetBucketLocation"
+    ]
+
+    resources = [
+      "arn:aws:rds:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:db:*",
```

Comment:
Ideally we would restrict this just to the pathfinder DB but unable to reference the db instanceId as it's not output by the RDS module.

Reply:
That's a good point. I've opened a ticket for us to add this as an output. I don't know when that will be done, so definitely carry on with this policy for now.

Reply:
Thanks David, I'll keep an eye for the change.

Reply:
@garethmdavies,  We have added db_identifier as an output to the latest RDS module. https://github.com/ministryofjustice/cloud-platform-terraform-rds-instance/blob/master/output.tf#L42. Pls have a look if that fix the issue here.